modules/planning/tasks/deciders/path_bounds_decider/path_bounds_decider.h

```diff
@@ -109,6 +109,17 @@ class PathBoundsDecider : public Decider {
       const ReferenceLineInfo& reference_line_info,
       std::vector<std::tuple<double, double, double>>* const path_bound);
 
+  std::string GeneratePullOverPathBound(
+      const ReferenceLineInfo& reference_line_info,
+      std::vector<std::tuple<double, double, double>>* const path_bound);
+
+  // bool SetUpPullOver(
```

Comment:
What are those commented lines for?  Future interface?

Reply:
Yes.